README.rst

```diff
@@ -2,7 +2,7 @@ InstrumentKit
 =============
 
 .. image:: https://img.shields.io/travis/Galvant/InstrumentKit/master.svg?maxAge=2592000
-    :target: https://travis-ci.org/Galvant/InstrumentKit
+    :target: https://travis-ci.com/Galvant/InstrumentKit
```

Comment:
Should probably be moved to GitHub actions now... however, I just noticed that the actions were not running on the main branch

Reply:
Wow my brain didn't even process that, you're right I'll update that

And yeah I noticed it wasn't running. I assumed it used the same rules but I guess I'm wrong :grimacing:

Reply:
I'm wondering if the $default-branch variable is the culprit... It works in [my case](https://github.com/galactic-forensics/iniabu/blob/main/.github/workflows/package_testing.yml) where I point out the default branch name as:

```yaml
on: 
  push:
    branches: [ main ]
```

Reply:
Yeah I think you're right. I was under the impression that this worked, but maybe I have a setting wrong somewhere. Not worth spending a bunch of time figuring out so I updated it

Reply:
Think this should work. One thing to think about, probably for another PR, is to move the default branch name to `main`, as is GitHub default now: https://github.com/github/renaming

Reply:
Agreed, I'm going to do that seperately